ValetTests/ValetTests.m

```diff
@@ -64,14 +67,21 @@ - (void)setUp;
 {
     [super setUp];
     
-    self.valet = [[VALValet alloc] initWithIdentifier:@"valet_testing" accessibility:VALAccessibilityWhenUnlocked];
-    self.testingValet = [[VALTestingValet alloc] initWithIdentifier:@"valet_testing" accessibility:VALAccessibilityWhenUnlocked];
-    self.synchronizableValet = [[VALSynchronizableValet alloc] initWithIdentifier:@"valet_testing" accessibility:VALAccessibilityWhenUnlocked];
+    NSString *const valetTestingIdentifier = @"valet_testing";
+    self.valet = [[VALValet alloc] initWithIdentifier:valetTestingIdentifier accessibility:VALAccessibilityWhenUnlocked];
+    self.testingValet = [[VALTestingValet alloc] initWithIdentifier:valetTestingIdentifier accessibility:VALAccessibilityWhenUnlocked];
+    self.synchronizableValet = [[VALSynchronizableValet alloc] initWithIdentifier:valetTestingIdentifier accessibility:VALAccessibilityWhenUnlocked];
+#if TARGET_OS_IPHONE && __IPHONE_8_0
```

Comment:
Would be nice to see some comments on why these macros.  also possbly maybe do a `#define VAL_IS_IPHONE_8_OR_GREATER (TARGET_OS_IPHONE && __IPHONE_8_0)` somewhere to make it so you're not repeating `TARGET_OS_IPHONE && __IPHONE_8_0` all over.